src/core/lib/iomgr/ev_apple.cc

```diff
@@ -39,14 +39,8 @@
 #include "src/core/lib/gprpp/time_util.h"
 #include "src/core/lib/iomgr/ev_apple.h"
 
-#ifndef NDEBUG
-#define GRPC_POLLING_TRACE(format, ...)                              \
-  if (GRPC_TRACE_FLAG_ENABLED(apple_polling)) {                      \
-    VLOG(2) << "(polling) " << absl::StrFormat(format, __VA_ARGS__); \
-  }
-#else
-#define GRPC_POLLING_TRACE(...)
-#endif  // NDEBUG
+#define GRPC_POLLING_TRACE() \
```

Comment:
nit: This macro should be deleted and replaced with `GRPC_TRACE_VLOG(apple_polling, INFO) << "(polling) " <<`. We can do that in another pass.

Reply:
Same as above . Changed the usage , but cant delete the macro - it messes up some old internal code which I will need to fix before I clean this up. Will do that in the next iteration.... This is going to be a long chain of PRs...